test/e2e/app-dir/typed-routes-validator/pages/api/test-route.ts

```diff
@@ -1,12 +1,11 @@
-import type { NextApiRequest, NextApiResponse } from 'next/types'
+import type { NextApiHandler } from 'next/types'
 
 type ResponseData = {
   message: string
 }
 
-export default function handler(
-  req: NextApiRequest,
-  res: NextApiResponse<ResponseData>
-) {
```

Comment:
Is that no longer valid code? If it's still valid, let's keep the test and add a new one instead. Unless there's an existing test for this pattern?

Reply:
Just restored the test in this PR!